html/php-pre-challenge1/index.php

```diff
@@ -1,5 +1,15 @@
 <?php
 for ($i = 1; $i <= 100; $i++) {
-    //ここからコードを書く
-    echo $i;
+    if ($i % 3 === 0 && $i % 5 === 0) {
```

Comment:
(修正不要)
```suggestion
    if ($i % 15 === 0) {
```
ゲームを作る場合などはこう書いて少しでも高速化させるようです。
業務アプリならむしろ３と５に分けた方が可読性が高くて好ましい気もしますね。
この辺は現場のチームのリーダに訊いてください。